ecommerce-ui/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+// import logo from './logo.svg';
+import './App.css';
+import Apartments from './Apartments';
```

Comment:
Good importing data a the top level component. Once you add state to hold shopping cart It will likely reside here as well.